packages/navigator/src/browser/navigator-frontend-module.ts

```diff
@@ -14,6 +14,8 @@ import { FileNavigatorContribution } from './navigator-contribution';
 import { createFileNavigatorWidget } from "./navigator-container";
 import { WidgetFactory } from '@theia/core/lib/browser/widget-manager';
 
+import '/style/index.css';
```

Comment:
The generated `.js` file will be put under `navigator/lib/browser/navigator-frontend-module.js` while the CSS file remains under `navigator/workspace/src/browser/style/index.css`. Hence you need to traverse up from the `lib` folder to the Theia extension root, then locate the CSS file with the `import '../../src/browser/style/index.css';` statement.

I believe, the incorrect CSS location causes the build failures currently.  See for example [here](https://travis-ci.org/theia-ide/theia/jobs/316740096#L1117). If you have questions, feel free to ask, I am happy to help you with the onboarding.

Reply:
Oh I see, thanks for the input. Will be making these changes asap.